packages/gen-ai/frontend/src/app/Chatbot/ChatbotMain.tsx

```diff
@@ -73,6 +79,22 @@ const ChatbotMain: React.FunctionComponent = () => {
     onShowErrorAlert: alertManagement.onShowErrorAlert,
   });
 
+  const isViewCodeDisabled = React.useMemo(() => !input || !modelId, [input, modelId]);
```

Comment:
There is no need to use `useMemo` here, you are creating a boolean, not an object. Also, the `isViewCodeDisabled` you are creating is not the dependency of any other variants.